src/common.ts

```diff
@@ -42,7 +32,7 @@ export function readJamsocketConfig(): JamsocketConfig | null {
 export function writeJamsocketConfig(config: JamsocketConfig): void {
   const dir = dirname(JAMSOCKET_CONFIG)
   mkdirSync(dir, { recursive: true })
-  writeFileSync(JAMSOCKET_CONFIG, `${JSON.stringify(config, null, 2)}${NEWLINE}`)
+  writeFileSync(JAMSOCKET_CONFIG, `${JSON.stringify(config, null, 2)}\n`)
```

Comment:
Will this work for Windows?

Reply:
Good catch. It wouldn't add the `\r`, I don't think that's a huge problem with modern Windows editors but I've replaced it with `EOL` from `os` which should do the trick.